scripts/README.md

```diff
@@ -0,0 +1,45 @@
+# Conversation AI Crowdsource (TypeScript) Scripts
+
+Various typescript scripts intended to be run with ts-node for Conversation AI
+crowdsource data management. General useage expected to be:
+
+```ts-node ts-bin/<nameofscript> <arguments>```
+
+## Installing Dependencies
+
+Global node dependencies for development: gcloud, node (suggest you use nvm to install it) are [typescript](https://www.typescriptlang.org/) to write sensible JS code, and [yarn](https://yarnpkg.com/lang/en/), and of course node (which is usually most easily installed and managed using [nvm](https://github.com/creationix/nvm/blob/master/README.md)):
+
+After you have installed node/npm using nvm, you can install the other global dependencies using:
+
+```bash
+npm install -g typescript yarn ts-node
+```
+
+Then from this directory, use yarn to install the local package dependencies:
+
+```bash
+yarn install
+```
+
+## Run Scripts
+
+```bash
```

Comment:
Maybe add a quick note about what running this command does?

Reply:
Good idea, also, the details are in the file, so I just wrote a comment about what this is intended to do, and pointed to the file for more details.